bruteforce/src/lib.rs

```diff
@@ -2,19 +2,11 @@
 
 #![crate_name = "bruteforce"]
 #![feature(
-    const_fn,
     test,
     generators,
-    generator_trait,
-    const_if_match,
-    const_panic,
     proc_macro_hygiene
 )]
 #![cfg_attr(not(feature = "std"), no_std)]
-
-extern crate test;
-
-#[cfg(not(feature = "std"))]
```

Comment:
Why do you want to remove this line?
We only want to replace `std` with `no_std_compat` if `no_std`.

Reply:
For me, it doesn't make much sense